### PR TITLE
US02-01: Background carousel

### PR DESCRIPTION
**Pull request by NEL45**

### CODE REVIEW

**Review by ChristianOlmedo — CHANGES_REQUESTED** [review]
layout file and too long title file need to be better indented

**Review by ChristianOlmedo — APPROVED** [review]

File: public/assets/css/welcome.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.load{
+    animation: slide 2s;
+}
+.slider{
+    margin:0;
+    padding: 0;
```
Comment (LucyMerlier) [inline public/assets/css/welcome.css:6]: magic values

File: public/assets/css/welcome.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 100vh;
```
Comment (LucyMerlier) [inline public/assets/css/welcome.css:11]: magic values

File: public/assets/css/welcome.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    }
+}
+
+/*Mobile version*/
+
+@media screen and (max-width: 670px){
+    .slider{
+        background-size: cover;
+    }
+}
```
Comment (LucyMerlier) [inline public/assets/css/welcome.css:74]: avoid magic values

File: src/View/User/Includes/_welcome_background_slideshow.html.twig [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{# User/Includes/_welcome_background_slideshow.html.twig #}
+
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+            </div>
+        </div>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_welcome_background_slideshow.html.twig:9]: empty divs?

File: src/View/User/welcome.html.twig [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+{% extends 'layout.html.twig' %}
+{% block stylesheet %}
+    <link rel="stylesheet" href= "/assets/css/welcome.css">
+{% endblock %}
+
+{% block content %}
+
+        {% include 'User/Includes/_welcome_background_slideshow.html.twig' %}
+    
+{% endblock %}
```
Comment (LucyMerlier) [inline src/View/User/welcome.html.twig:11]: indentation? empty lines?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: public/assets/css/welcome.css (hunk 2 above)
Comment (NEL45) [reply public/assets/css/welcome.css:11]: correction ok

File: public/assets/css/welcome.css (hunk 3 above)
Comment (NEL45) [reply public/assets/css/welcome.css:74]: magic values modified

File: src/View/User/welcome.html.twig (hunk 5 above)
Comment (NEL45) [reply src/View/User/welcome.html.twig:11]: comment including

File: src/View/User/Includes/_welcome_background_slideshow.html.twig (hunk 4 above)
Comment (NEL45) [reply src/View/User/Includes/_welcome_background_slideshow.html.twig:9]: these voids are provided for a card already created

**Pull request closed without merging**